### PR TITLE
chore(i18n,client): Adding meta-tags.json, links.json and motivation.json in German

### PR DESCRIPTION
**Pull request by stephanduesterhoeft**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by Sboonny** [comment]
Translation is passed through crowdin by camperbot. I am closing this because it isn't the right approach to translating.

---

If you have any questions, feel free to ask questions on the ['Contributors' category on our forum](https://forum.freecodecamp.org/c/contributors) or [the contributors chat room](https://discord.gg/PRyKn3Vbay).

**Comment by stephanduesterhoeft** [comment]
@Sboonny @ieahleen 

I honestly don't understand that. This was discussed with Ilenia. The meta-tag.json and motivation.json are not stored in Crowdin for translation!?

**Comment by Sboonny** [comment]
If I did a mistake, this can simply be reopened.

**Comment by ieahleen** [comment]
@Sboonny these are the files that are *not* translated through Crowdin.

**Comment by ieahleen** [comment]
@stephanduesterhoeft if you can fix the conflicts, which I think will just be telling git that your files are to keep, as they have been created as copy of the english ones through #47312 this can be reviewed

**Comment by stephanduesterhoeft** [comment]
Hello @ieahleen. No idea how to do this. I will adjust tonight the origin files now in the folder German and make a new PR.

**Comment by ieahleen** [comment]
@stephanduesterhoeft you don't need to open a new PR, you can just do it here, or through github interface or your local instance
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts

**Comment by stephanduesterhoeft** [comment]
@ieahleen Thanks a lot for your help! I will take care of it.

File: client/i18n/locales/german/meta-tags.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
...
-    "learn",
+    "lernen",
     "tutorial",
-    "programming"
+    "programmierung"
   ],
-  "youre-unsubscribed": "You have been unsubscribed"
-}
+  "youre-unsubscribed": "Du wurdest abgemeldet"
+}
```
Comment (ieahleen) [inline client/i18n/locales/german/meta-tags.json:32]:
```suggestion
}

```
do not remove the last new line

File: client/i18n/locales/german/motivation.json [hunk 2]
```diff
@@ -1,819 +1,203 @@
...
-      "quote": "There is no elevator to success, you have to take the stairs.",
-      "author": "Zig Ziglar"
-    },
-    {
-      "quote": "You don't have to be great to start, but you have to start to be great.",
-      "author": "Zig Ziglar"
-    }
   ]
-}
+}
```
Comment (ieahleen) [inline client/i18n/locales/german/motivation.json:203]: same here, don't remove the last new line
```suggestion
}

```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
A couple of things to fix

File: client/i18n/locales/german/motivation.json (hunk 2 above)
Comment (stephanduesterhoeft) [reply client/i18n/locales/german/motivation.json]: @ieahleen Thank you for your advice and help!

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @stephanduesterhoeft, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Pull request merged**